readme.md

```diff
@@ -691,6 +691,7 @@
 ## Decentralized Systems
 
 - [Bitcoin](https://github.com/igorbarinov/awesome-bitcoin#readme) - Bitcoin services and tools for software developers.
+- [TON (Open Network)](https://github.com/ton-community/awesome-ton#readme) - A pioneering blockchain platform from Telegram, TON combines speed, security, and usability, facilitating next-gen cryptocurrency transactions.
```

Comment:
The entry must be moved to the bottom of "Decentralized Systems" as stated in the guidelines: `Your entry should be added at the bottom of the appropriate category.`

Reply:
Thank you I have submitted the changes

https://github.com/sindresorhus/awesome/pull/2655/commits/cfa80c65c04103b9b8fbd8510cf48627f210065d